stabbity.lic

```diff
@@ -0,0 +1,253 @@
+=begin
+Stabbity, Stabbity, Stabbity. Repeat.
```

Comment:
Please link to elanthipedia and post relevant documentation there.

Reply:
Documentation added to elanthipedia and link added to script header in next version: https://elanthipedia.play.net/Lich_script_repository#stabbity.